core/src/main/java/io/aiven/kafka/tieredstorage/chunkmanager/DiskBasedCachingChunkManagerConfig.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.tieredstorage.chunkmanager;
+
+import java.nio.file.Path;
+import java.util.Map;
+
+import org.apache.kafka.common.config.ConfigDef;
+
+public class DiskBasedCachingChunkManagerConfig extends CachingChunkManagerConfig {
```

Comment:
Could we describe here why do we need 2 paths?

Reply:
Was discussed before and was going to address this later but done already.